### DISCUSSION TITLE
Dashboard/Panel show/hide according to run time evaluations

### DISCUSSION THREAD

**Discussion by chavalikesavanand** [grafana/grafana#44202] [Dashboards, Panels & Visualizations]
We are using dashboard links to list all dashboards available for a target, but there is no way to hide some of them based on some target property.
For example for dashboards Like HyperV, Oracle and others that are dependent on availability of certain metrics like whether HyperV is installed on the system or Oracle/MSSQL is installed. These dashboards should be shown only when they are relevant.
Same applies to panels in a dashboard (eg. we have a panel for read-only storage and a panel for free percentage... it would be nice to hide the second in the case that the first reports no read-only storage) 
Additionally, some dashboards (e.g. basic CPU, RAM, host information) will be differentiated between Linux and Windows targets
Let's imagine that we have a panel listing all targets in a table and, for each target, we wish to have an hyperlink in the target name to redirect the user to the main dashboard for that target
At the date we can't redirect to two different dashboards according to the fact that it is a Linux or windows machine

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
